docs/operation/cluster_and_service_configuration.rst

```diff
@@ -5,72 +5,291 @@ This section contains information describing the list of available Cluster and
+
+Dex Default Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Dex is an Identity Provider that drives user authentication and identity
+management in a MetalK8s cluster.
 
-This document below describes parameters that are set by default.
+The default configuration values for Dex are specified below:
 
 .. literalinclude:: ../../salt/metalk8s/addons/dex/config/dex.yaml
 
-Grafana
-"""""""
-The default configuration values for Grafana are specified in the output below
-and can be overridden by specifying its corresponding values within the Cluster
-and Service ConfigMap (metalk8s-grafana-config).
+See :ref:`csc-dex-customization` for Dex configuration customizations.
 
-This document below describes parameters that are set by default.
+Grafana Default Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Grafana is a web interface used to visualize and analyze metrics scraped by
+Prometheus, with nice graphs.
+
+The default configuration values for Grafana are specified below:
 
 .. literalinclude:: ../../salt/metalk8s/addons/prometheus-operator/config/grafana.yaml
 
-Prometheus
-""""""""""
-The default configuration values for Prometheus are specified in the output
-below and can be overridden by specifying its corresponding values within the
-Cluster and Service ConfigMap (metalk8s-grafana-config).
+.. _csc-prometheus-default-configuration:
+
+Prometheus Default Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This document below describes parameters that are set by default.
+Prometheus is responsible for monitoring all the applications and systems
+in the MetalK8s cluster.
+It scrapes and stores various metrics from these systems and then analyze them
+against a set of alerting rules.
+If a rule matches, Prometheus sends an alert to Alertmanager.
+
+The default configuration values for Prometheus are specified below:
 
 .. literalinclude:: ../../salt/metalk8s/addons/prometheus-operator/config/prometheus.yaml
 
+Service Configurations Customization
+------------------------------------
+
+.. _csc-alertmanager-customization:
+
+Alertmanager Configuration Customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Default configuration for Alertmanager can be overridden by editing its
+Cluster and Service ConfigMap ``metalk8s-alertmanager-config`` in namespace
+``metalk8s-monitoring`` under the key ``data.config\.yaml``:
+
+  .. code-block::
+
+     kubectl --kubeconfig /etc/kubernetes/admin.conf \
+       edit configmap -n metalk8s-monitoring metalk8s-alertmanager-config
+
+The following documentation is not exhaustive and is just here to give
+some hints on basic usage, for more details or advanced
+configuration, see the official `Alertmanager documentation`_.
+
+.. _Alertmanager documentation: https://prometheus.io/docs/alerting/configuration/
+
+Adding inhibition rule for an alert
+"""""""""""""""""""""""""""""""""""
+
+Alert inhibition rules allow to make one alert inhibit notifications for some
+other alerts.
```

Comment:
`Alert inhibition rules allow making one alert inhibit notifications for some other alerts.`